src/python/analyzer_executor/src/analyzer_executor_lib/analyzer_executor.py

```diff
@@ -171,32 +142,25 @@ def exec_analyzers(
         for an_name, queries in querymap.items():
             analyzer = analyzers[an_name]
 
-            for i, query in enumerate(queries):
+            for _, query in enumerate(queries):
```

Comment:
IF we aren't using 'i' we should just remove the enumerate entirely. This is another case of an optimization that was removed.